### PR TITLE
Add basic CI workflow that compiles the example(s)

### PR DESCRIPTION
**Pull request by rleh**

### CODE REVIEW

**Review by salkinium — APPROVED** [review]
Thanks!

File: .github/workflows/compile_examples.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Compile Examples
+
+on: [pull_request]
+
+jobs:
+  unittests-linux-generic:
```
Comment (rleh) [inline .github/workflows/compile_examples.yml:6]: Fail 😆

image

Comment (salkinium) [reply .github/workflows/compile_examples.yml:6]: I also missed that you requested a review from @chris-durand and just merged it… oops

Comment (salkinium) [reply .github/workflows/compile_examples.yml:6]: Force push?

Comment (rleh) [reply .github/workflows/compile_examples.yml:6]: Nah, the CI job name is not that important to justify a force push.

**Pull request merged**